src/arch/helperneon32.h

```diff
@@ -124,16 +168,18 @@ static INLINE vfloat vsqrt_vf_vf(vfloat d) {
   return (float32x4_t)vbicq_u32((uint32x4_t)u, vceqq_f32(d, vdupq_n_f32(0.0f)));
 }
 
+static INLINE vfloat vrec_vf_vf(vfloat d) {
+  float32x4_t x = vrecpeq_f32(d);
+  x = vmulq_f32(x, vrecpsq_f32(d, x));
+  return vmlsq_f32(vaddq_f32(x, x), vmulq_f32(x, x), d);
+}
+
 static INLINE vfloat vrecsqrt_vf_vf(vfloat d) {
   float32x4_t x = vrsqrteq_f32(d);
   x = vmulq_f32(x, vrsqrtsq_f32(d, vmulq_f32(x, x)));
   return vmlaq_f32(x, vmlsq_f32(vdupq_n_f32(1), x, vmulq_f32(x, d)), vmulq_f32(x, vdupq_n_f32(0.5)));
 }
-
-static INLINE vfloat vabs_vf_vf(vfloat f) { return vabsq_f32(f); }
-static INLINE vfloat vneg_vf_vf(vfloat f) { return vnegq_f32(f); }
-static INLINE vfloat vmla_vf_vf_vf_vf(vfloat x, vfloat y, vfloat z) { return vmlaq_f32(z, x, y); }
-static INLINE vfloat vmlanp_vf_vf_vf_vf(vfloat x, vfloat y, vfloat z) { return vmlsq_f32(z, x, y); }
+#endif
```

Comment:
Please modify this line to `#endif /* #if CONFIG == 4 */` (assuming I got the right `#if`!!).